apps/deploy/upload-dapp-chain/src/configs/base.config.ts

```diff
@@ -0,0 +1,32 @@
+export default () => ({
+  app: {
+    PORT: process.env.PORT || "3000",
+  },
+  github: {
+    GITHUB_ACCESS_TOKEN: process.env.GITHUB_ACCESS_TOKEN || "token",
+    GITHUB_OWNER_REPO: process.env.GITHUB_OWNER_REPO || "owner",
+    GITHUB_API_BASE_URL: process.env.GITHUB_OWNER_REPO || "baseUrl",
+  },
+  bot: {
+    TELEGRAM_BOT_TOKEN: process.env.TELEGRAM_BOT_TOKEN || "bot",
+  },
+  workflow: {
+    WORKFLOW_PATH: process.env.WORKFLOW_PATH || "path",
+  },
+  nftMarketplace: {
+    NFT_MARKETPLACE_ADMIN_ID: process.env.NFT_MARKETPLACE_ADMIN_ID || "nftMarketplaceId",
+    NFT_MARKETPLACE_TREASURY_ID: process.env.NFT_MARKETPLACE_TREASURY_ID || "treasuryId",
+  },
```

Comment:
This is unnecessary in environment variables

Reply:
Deleted